spanner_orm/tests/models.py

```diff
@@ -126,10 +126,13 @@ class UnittestModelWithoutSecondaryIndexes(model.Model):
   float_2 = field.Field(field.Float, nullable=True)
   string = field.Field(field.String, primary_key=True)
   string_2 = field.Field(field.String, nullable=True)
+  string_3 = field.Field(field.String, nullable=True, length=20)
   bytes_ = field.Field(field.BytesBase64, primary_key=True)
   bytes_2 = field.Field(field.BytesBase64, nullable=True)
+  bytes_3 = field.Field(field.BytesBase64, nullable=True, length=20)
   timestamp = field.Field(field.Timestamp)
   string_array = field.Field(field.StringArray, nullable=True)
+  string_array_2 = field.Field(field.StringArray, nullable=True, length=20)
```

Comment:
(nit) The docstring for this class is no longer accurate, please add these fields to UnittestModel too.